<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetClass</class>
 <widget class="QWidget" name="QtWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtWidget</string>
  </property>
  <widget class="QPushButton" name="infoBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </widget>
  <widget class="QPushButton" name="warningBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Warning</string>
   </property>
  </widget>
  <widget class="QPushButton" name="errorBtn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>150</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Error</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtPopupWidget.qrc"/>
 </resources>
 <connections/>
</ui>
